third_party/mkl_dnn/build_defs.bzl

```diff
@@ -14,18 +14,16 @@ def if_mkl_open_source_only(if_true, if_false = []):
         "//conditions:default": if_false,
     })
 
-def if_mkldnn_threadpool(if_true, if_false = []):
-    """Returns `if_true` if MKL-DNN v1.x is used.
-
-    Shorthand for select()'ing on whether we're building with
-    MKL-DNN v1.x open source library only with user specified threadpool, without depending on MKL binary form.
+def if_mkldnn_openmp(if_true, if_false = []):
```

Comment:
Should we change this to `if_onednn_openmp` instead? We can do this in a later. I'm also fine with keeping `mkldnn` if you prefer.

Reply:
@penpornk I left it as mkldnn as there are many other functions still using mklnn and mkl in the build files.  I am planning to submit a PR to cleanup all the build files and rename uses of mkldnn and mkl.

Reply:
Sounds good to me.